.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 build
+.idea
```

Comment:
This shouldn't be in the same pull request as a proposed code change, a separate pull request "if any".

This really should be added to your global `.gitignore`, there are too many IDEs and Editors to accomadate, see https://core.trac.wordpress.org/changeset/35254